gitlab/resource_gitlab_service_jira.go

```diff
@@ -0,0 +1,173 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabServiceJira() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabServiceJiraCreate,
+		Read:   resourceGitlabServiceJiraRead,
+		Update: resourceGitlabServiceJiraUpdate,
+		Delete: resourceGitlabServiceJiraDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceGitlabServiceJiraImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"title": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"updated_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"active": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"url": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateURLFunc(),
+			},
+			"project_key": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "",
+			},
+			"username": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"password": {
+				Type:      schema.TypeString,
+				Required:  true,
+				Sensitive: true,
+			},
+			"jira_issue_transition_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+		},
+	}
+}
+
+func resourceGitlabServiceJiraCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*gitlab.Client)
+
+	project := d.Get("project").(string)
+
+	jiraOptions, err := expandJiraOptions(d)
+	if err != nil {
+		return err
+	}
+
+	log.Printf("[DEBUG] Create Gitlab Jira service")
+
+	_, setServiceErr := client.Services.SetJiraService(project, jiraOptions)
+	if err != nil {
+		return fmt.Errorf("[ERROR] Couldn't create Gitlab Jira service: %s", setServiceErr)
+	}
+
+	d.SetId(project)
+
+	return resourceGitlabServiceJiraRead(d, meta)
+}
+
+func resourceGitlabServiceJiraRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*gitlab.Client)
+	project := d.Get("project").(string)
+
+	log.Printf("[DEBUG] Read Gitlab Jira service %s", d.Id())
+
+	jiraService, response, err := client.Services.GetJiraService(project)
+	if err != nil {
+		if response.StatusCode == 404 {
```

Comment:
please remove this

Reply:
@jorcau mmh can you do this in a new pr?

Reply:
Done in #147